src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html

```diff
@@ -144,7 +144,8 @@
               More
             </button>
             <div id="moreOptionsDropdownMenu" ngbDropdownMenu aria-labelledby="moreOptionsDropdownMenuButton">
-              <a class="dropdown-item" (click)="markAsStarred()" [translate]="'mail_list.add_star'">Add star</a>
+              <a *ngIf="isNeedAddStar()" class="dropdown-item" (click)="markAsStarred(true)" [translate]="'mail_list.add_star'">Add star</a>
+              <a *ngIf="isNeedRemoveStar()" class="dropdown-item" (click)="markAsStarred(false)" [translate]="'mail_list.remove_start'">Remove star</a>
```

Comment:
`[translate]="'mail_list.remove_start'">Remove star`

Should it say `remove_star`?

Reply:
Yes, exactly
I know the original has typo, I will make the update